device/celestica/x86_64-cel_silverstone-r0/led_proc_init.soc

```diff
@@ -0,0 +1,12 @@
+#Enable all ports
+sleep 6
+port all en=1
+#sleep 6
+#linkscan 250000; port xe,ce linkscan=on
+
+#Load LED
```

Comment:
Looks like LED code is commented out, can you make a comment here why and what to expect for the fix?

Reply:
Just now the port LED can't work in TH3.  In TH3, the LED FW must be uploaded after cancun FW loading, it will take effect, otherwise, it does't work, We have submit a PR to Broadcom to fix the issue in SAI.

m0 load 0 0x0 /usr/share/sonic/platform/linkscan_led_fw.bin
m0 load 0 0x3800 /usr/share/sonic/platform/custom_led.bin

Reply:
Please add a comment in the code itself, so we can fix it later once the SAI is fixed.

Reply:
fixed in https://github.com/Azure/sonic-buildimage/pull/3474/commits/2294129ca65505c513cce98386a796a7af3373ed